package-lock.json

```diff
@@ -6606,6 +6639,21 @@
         }
       }
     },
+    "simple-swizzle": {
```

Comment:
I don't think this is used? In that case it should be removed

Reply:
Is this used or did you forget to remove it? :) @fredriksiemund

Reply:
Saw now that react-native-svg is dependent on color, color-string and simple-swizzle. Should I still remove them?